src/ui/UserListItem/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const getGlobalUserTag = (metadata: string) => {
+  return metadata === 'coach' ? 'Coach Emocional' : metadata === 'psychologist' ? 'Psicólogo/a' : metadata;
```

Comment:
Customer support case is missing

Reply:
Les dejé el nombre de Atención al cliente en el campo del dashboard de sendbird directamente